recipes/automake/all/test_v1_package/conanfile.py

```diff
@@ -0,0 +1,90 @@
+from conan import ConanFile
+from conans import AutoToolsBuildEnvironment, tools
+from conan.tools.microsoft import is_msvc
+from contextlib import contextmanager
+import os
+import shutil
+
+required_conan_version = ">=1.45.0"
```

Comment:
Also bump to `1.51.x` for consistency?

Reply:
Technically this recipe could be used with an older Conan version, in reality it won't. Because there is such a tight coupling. I will update it. But I have a suspicion that the versions between recipe and test_package will start to drift when ever the recipe is updated again for new functionality